persistence/sql/persister_identity.go

```diff
@@ -83,7 +83,7 @@ func (p *Persister) normalizeIdentifier(ct identity.CredentialsType, match strin
 	return match
 }
 
-func (p *Persister) FindByCredentialsIdentifier(ctx context.Context, ct identity.CredentialsType, match string) (*identity.Identity, *identity.Credentials, error) {
+func (p *Persister) FindByCredentialsIdentifier(ctx context.Context, match string) (*identity.Identity, error) {
```

Comment:
I didn't forget the normalization 😓 I left it out on purpose because the credential type isn't known before the query. I left a note in the github issue about that earlier - https://github.com/ory-corp/cloud/issues/3947#issuecomment-1422394212